surveys/2023-annual-survey/questions.md

```diff
@@ -0,0 +1,768 @@
+- Debugging experience
+- Available tools and support
+- Async programming
+- GUI development
+- Rust language and standard library documentation
+
+> **justification**
+>
+> This question gathers data on the communities perceptions of certain aspects of Rust at this point in time.
+
+### Do you agree with the following statements on Rust stability?
+
+Type: matrix (optional)
+
+Statements:
+
+- I can upgrade the *stable* compiler version without fear of my code failing to compile
+- I can upgrade the *nightly* compiler version without fear of my code failing to compile
+- Upgrading to a new *stable* compiler version requires either no changes or extremely small & easy changes to my code
+- Upgrading to a new *nightly* compiler version requires either no changes or extremely small & easy changes to my code
+
+Rating:
+
+- Agree
+- Disagree
+
+> **justification**
+>
+> When want to get an impression of how stable the compiler *feels*. Impressions are more important than hard numbers as
+> not all users define stability in the same way the compiler does.
+
+### Do you agree with the following statements on Rust employment?
+
+Type: matrix (optional)
+
+Statements:
+
+- It is easy for qualified applicants to find jobs which use Rust for the majority of programming
+- Existing Rust jobs are attractive
+
+Rating:
+
+- Agree
+- Neither agree nor disagree
+- Disagree
+
+> **justification**
+>
+> The flip side of the question asking whether the respondent's company plans on hiring Rust developers, we
+> want to know how respondents feel the level of demand for and quality of Rust jobs are.
+
+## About you
+
+See [who](./design/who.md).
+
+The following are primarily for cohort analysis, secondarily for understanding the shape of the community.
+
+For methodological purposes, the bulk of the demographics should be at the end of the survey (unless acting as filter/flow questions above).
+They're both easy to complete (beneficial at the end) and somewhat personal (but at this point folks are invested and we've built 'trust').
+Can also be problematic at start if we're asking all easy, personal questions and then get to the harder ones - easy to drop out.
+
+### Do you consider yourself a member of a group which is underrepresented or marginalized in technology?
+
+Please share only what you are comfortable sharing. This will help us better serve underrepresented and marginalized groups, better understand how well our outreach efforts are going, and more.
+
+Type: select one
+
+- Yes
+- No
+- I prefer not to say
+
+### Do you consider yourself part of any of the following underrepresented or marginalized groups?
+
+Please share only what you are comfortable sharing. This will help us better serve underrepresented and marginalized groups, better understand how well our outreach efforts are going, and more.
+
+Type: select all that apply (optional)
+
+- Cultural beliefs
+- Disabled or person with disability (including physical, mental, and other)
```

Comment:
I gave a thoughtful read of #232, there's a lot to unpack in there. I've tried understanding the suggestions and the various point of view. I'd like to close that issue by proposing this change:
```suggestion
- Disabled (including physically, mentally or other)
- Neurodivergent
```

Reply:
I am also pinging some people that commented there in an effort to have a second opinion from _sensitivity readers_ mentioned in that issue.

@yaahc @workingjubilee wdyt?

Reply:
It's been a while since I participated in that thread and I haven't gone back and fully re-reviewed it, but looking at the changes this seems like a great start.